extensions/wikia/AdEngine/js/context/adContext.js

```diff
@@ -80,12 +87,7 @@ define('ext.wikia.adEngine.adContext', [
 		}
 
 		// PageFair recovery
-		if (!noExternals &&
-			context.opts.pageFairRecovery !== false &&
-			geo.isProperGeo(instantGlobals.wgAdDriverPageFairRecoveryCountries))
-		{
-			context.opts.pageFairRecovery = true;
-		}
+		context.opts.pageFairRecovery = isPageFairRecoveryEnabled(noExternals, instantGlobals);
```

Comment:
I'm not sure about this change since it's not consistent with how are set other context vars